onnxruntime/core/providers/partitioning_utils.h

```diff
@@ -62,12 +65,13 @@ Create the supported partitions for the execution provider.
 std::vector<std::unique_ptr<ComputeCapability>>
 CreateSupportedPartitions(const GraphViewer& graph_viewer,
                           const IsNodeSupportedFn& is_node_supported_fn,
+                          const GetNodeUnitFn& get_node_unit_fn,
                           const OnGroupClosedFn& on_group_closed_fn,
```

Comment:
I'm hoping this goes away as I'm still not convinced the partitioning utils need to know or care about whether something is in a QDQ group or not.

Reply:
This was added previously and should be removed.